saleor/settings.py

```diff
@@ -532,6 +532,9 @@ def SENTRY_INIT(dsn: str, sentry_opts: dict):
     ],
 }
 
+# Set GRAPHQL_QUERY_MAX_COMPLEXITY=0 in env to disable (not recommended)
+GRAPHQL_QUERY_MAX_COMPLEXITY = int(os.environ.get("GRAPHQL_QUERY_MAX_COMPLEXITY", 0))
```

Comment:
Since 0 is not recommended, we could set it to some meaningful value by default (probably the same value that would be set in the cloud anyway).

Reply:
It's too early for that. Devs should first set costs on some of fields based on known usage patterns and then set default query limit.

Reply:
OK sounds good.